data/political/2012-2022-political-districts/index.html

```diff
@@ -1,146 +0,0 @@
----
```

Comment:
**chore**: undelete and mark as publish: false

Reply:
is there an easy way to do this, aka: remove the associated commit or should i just add the page back into this branch?

Reply:
removing the commit would be the easiest way

Reply:
Is there an easy way to remove a commit?  I'm seeing that option in the browser.